### ISSUE TITLE
Error with v1.1.6 Get-ACMEServiceDirectory: Cannot convert the "C:\Temp" value of type "System.String" to type "AcmeState"

### ISSUE THREAD

**Issue by andyghc** [PKISharp/ACMESharpCore-PowerShell#77]
Hi,

I am noticing a problem with v1.1.6 when first setting up the account.

`$acmeServiceName = "LetsEncrypt-Staging";
$acmeStateDir = "C:\Temp;`
`New-ACMEState -Path $acmeStateDir;`
`Get-ACMEServiceDirectory -State $acmeStateDir -ServiceName $acmeServiceName;`

Which returns this:

`Failed
Cannot process argument transformation on parameter 'State'. Cannot convert the "C:\Temp" value of type "System.String" to type "AcmeState". (Cannot convert the "C:\Temp" value of type "System.String" to type "AcmeState". (Cannot convert the "C:\Temp" value of type "System.String" to type "AcmeState".))`

I can confirm that simply importing the previous version (1.1.5) and running the same commands does not error.  (Import-Module -name "ACME-PS" -MaximumVersion "1.1.5")

Is there an introduced bug or am I doing something wrong?  NB I first noticed this when copying my Azure Automation script to a new Automation Account, which imported the latest version of ACME-PS.  I then reproduced the error on my PC contrasting the two versions.

**Comment by glatzert** [comment]
I assume the error happens in Get-ACMEServiceDirectory?

Could you please provide the output of $PSVersionTable

**Comment by andyghc** [comment]
Hi,

Yes, it errors in the Get-ACMEServiceDirectory command, but only in v1.1.6

```
PS C:\WINDOWS\system32> $PSVersionTable

Name                           Value
----                           -----
PSVersion                      5.1.17763.1007
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17763.1007
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

**Comment by glatzert** [comment]
I just released 1.1.7, fixing that bug - I locally validated it loading the state.
Sorry - I thought I tested it thoroughly, but missed the simplest case.

**Comment by andyghc** [comment]
Confirming... the ACME-PS v1.1.7 module now works as expected.  Thanks.
